### ISSUE TITLE
custom dEntities no longer get recognised by scriptname after chunk unloads

### ISSUE THREAD

**Issue by BlackCoyote** [DenizenScript/Denizen-For-Bukkit#1209]
`- spawn e@scriptname <location>`

```
on scriptname damaged:
- narrate "oink"
```

you get the idea here

When the chunk gets unloaded, and you load it back in to go to the entity, it no longer gets recognised in those events when the entity is visibly still there.

Tested on dev 390.
Tested on multiple servers besides my own infamous one (don't blame my server mcmonkey!).

**Comment by BlackCoyote** [comment]
mcmonkey pls

**Comment by mcmonkey4eva** [comment]
Can you give me a better way to test this so I can know for sure a fix works?

**Comment by Fortifier42** [comment]
I.. can't replicate this

**Comment by mcmonkey4eva** [comment]
@Fortifier42 I couldn't either, thus my request for a better replication method, assuming he's doing something differenty that we're not thinking of. It could be another issue with just his server :o

**Comment by BlackCoyote** [comment]
"Tested on multiple servers besides my own infamous one (don't blame my server mcmonkey!)."

**Comment by BlackCoyote** [comment]
nice reading m8

**Comment by Fortifier42** [comment]
How about a proper replication method?

**Comment by BlackCoyote** [comment]
http://mcmonkey.org/haste/22023 these are some snippets of script that it happens with

**Comment by Fortifier42** [comment]
Try setting the other stuff such as name in the script itself, not in the entity.

**Comment by BlackCoyote** [comment]
why would i

**Comment by BlackCoyote** [comment]
replicated with
```death_entity:
  type: entity
  entity_type: e@skeleton```

**Comment by BlackCoyote** [comment]
@mcmonkey4eva if you need to test a fix i'll happily do it for you as i already can replicate the problem

**Comment by mcmonkey4eva** [comment]
Fixed in 1.14+ by the tracker rewrite in https://github.com/DenizenScript/Denizen-For-Bukkit/commit/7872f0e8853ee24ebf00f4f1ab07de589a6a2d99
